internal/handler/dataBinder.go

```diff
@@ -3,19 +3,20 @@ package handler
 import (
 	"fmt"
 
-	"github.com/dhis2-sre/im-manager/internal/apperror"
+	"github.com/dhis2-sre/im-manager/internal/errdef"
+
 	"github.com/gin-gonic/gin"
 )
 
 func DataBinder(c *gin.Context, req interface{}) error {
 	if c.ContentType() != "application/json" && c.ContentType() != "multipart/form-data" {
-		reason := fmt.Sprintf("%s only accepts content of type application/json or multipart/form-data", c.FullPath())
-		return apperror.NewUnsupportedMediaType(reason)
+		message := fmt.Sprintf("%s only accepts content of type application/json or multipart/form-data", c.FullPath())
+		return errdef.NewBadRequest(message)
 	}
 
 	if err := c.ShouldBind(req); err != nil {
-		message := fmt.Sprintf("Error binding data: %+v\n", err)
-		return apperror.NewBadRequest(message)
+		message := fmt.Sprintf("Error binding data: %+v", err)
+		return errdef.NewBadRequest(message)
```

Comment:
I think you overlooked what I actually meant, unless I was wrong 😋. I think the call to fmt.Sprintf is redundant and should be removed 😄 

```suggestion
		return errdef.NewBadRequest("Error binding data: %+v", err)
```

Reply:
Yeah, the fmt.Sprintf should be removed but further more I think it make sense to remove the `UnsupportedMediaType` error type as well. By the end of the day, not posting the right content type is a bad request.

Reply:
yes which is what I already advocated for in comment https://github.com/dhis2-sre/im-manager/pull/259/files/3ded1ea09b64d95b2592c706a165a349916ce02d#r1214145209 the only thing thats left is removing the redundant fmt.Sprintf call

Reply:
Already done, I'm not sure why it's not reflected here